index.js

```diff
@@ -398,10 +403,6 @@
   var UNKNOWN       = 'UNKNOWN';
   var VARIABLE      = 'VARIABLE';
 
-  //  Inconsistent :: Type
-  var Inconsistent =
-  new _Type(INCONSISTENT, '', '', always2('???'), K(false), [], {});
-
```

Comment:
What's the thinking behind moving this definition? Did it feel inconsistent having it apart from the others?